pkg/kdump/kdump.scss

```diff
@@ -19,9 +19,9 @@
 
 @use "page";
 
-@import "global-variables";
-@import "@patternfly/patternfly/utilities/Text/text.scss";
-@import "@patternfly/patternfly/utilities/Spacing/spacing.scss";
+@use "global-variables" as *;
```

Comment:
Some numbers for the "I have not checked this" commit message:

| NODE_ENV | main | PR |
| ---------------- | ---------- | ---------- |
| development | 33MB | 33 MB |
| production | 2.6 MB | 2.7 MB | 

So it's by and large the same up to some noise. As an example file:
```
1648023 Mar 26 14:03 dist.main.devel/systemd/overview.css
1644008 Mar 26 14:09 dist.pr.devel/systemd/overview.css
 131976 Mar 26 14:04 dist.main.prod/systemd/overview.css.gz
 131914 Mar 26 14:08 dist.pr.prod/systemd/overview.css.gz
```